db/migrate/20181215225757_create_items.rb

```diff
@@ -6,7 +6,7 @@ def change
       t.string :thumbnail
       t.integer :price
       t.integer :inventory
-      t.boolean :enabled, default: false, null: false
+      t.boolean :enabled, default: true, null: false
```

Comment:
Big change, should get the whole team to approve @stoic-plus @le3ah. I'm fine with this change.